### ISSUE TITLE
Help wanted: Create datasets for coin images from other countries

### ISSUE THREAD

**Issue by EdjeElectronics** [EdjeElectronics/TensorFlow-Lite-Object-Detection-on-Android-and-Raspberry-Pi#153]
**Help Wanted!**

The example training dataset in the [Google Colab tutorial](https://colab.research.google.com/github/EdjeElectronics/TensorFlow-Lite-Object-Detection-on-Android-and-Raspberry-Pi/blob/master/Train_TFLite2_Object_Detction_Model.ipynb) uses images of coins from the United States. I like using coins as an example, because it's easy for anyone to find some spare change to test their newly trained detector with. However, I know there people from around the world who use this guide, and I'd like to add datasets of coins from other countries.

Want to help out by taking pictures of coins from their own countries, labeling them, and sending them to me? The images and labels can be zipped and shared with me using a Dropbox, Google Drive, or similar download link. Either email the link to me at evan.juras@ejtech.io or add it as a comment on this issue. I'll upload the images to Dropbox and add them to the list of download links in the ["Option 3. Use coin detection dataset" section](https://colab.research.google.com/github/EdjeElectronics/TensorFlow-Lite-Object-Detection-on-Android-and-Raspberry-Pi/blob/master/Train_TFLite2_Object_Detction_Model.ipynb#scrollTo=9xAJMKwpFilm).

**Image Requirements**

- Images must be taken at a similar distance and perspective as the ones in [my dataset](https://www.dropbox.com/s/gk57ec3v8dfuwcp/CoinPics_11NOV22.zip) (camera about 25-35cm above the table and looking straight down at the coins)
- Each coin must be clearly visible in each image
- Each coin must be clearly and accurately labeled using [LabelImg](https://github.com/heartexlabs/labelImg) with annotation data in PascalVOC format
- Please send a minimum of 100 labeled images

**Images Gathered So Far**
Let's work together to make an international coin image dataset! Here is the size of the dataset we've collected for each country so far:

- [USA: 750 images](https://www.dropbox.com/s/gk57ec3v8dfuwcp/CoinPics_11NOV22.zip)
